segregate_pos_neg_nos.py

```diff
@@ -0,0 +1,19 @@
+# Implemented using Bubble Sort
+
+ar = list(map(int, input("Enter the elements: ").split()))
```

Comment:
Can you move both these files to their own folders in the unclassified algorithms directory.

Reply:
Should I put them in src

Reply:
Yes.

Reply:
I put it in unclassified algorithms. How do I change them to their respective directories?

Reply:
Made the changes, please review.